namespaces/live.cloud-platform.service.justice.gov.uk/make-recall-decision-preprod/08-redirect_old_url.yaml

```diff
@@ -0,0 +1,28 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  annotations:
+    nginx.ingress.kubernetes.io/rewrite-target: https://consider-a-recall-preprod.hmpps.service.justice.gov.uk/$1
+    external-dns.alpha.kubernetes.io/set-identifier: make-recall-decisions-redirect-old-url-make-recall-decision-preprod-green
+    external-dns.alpha.kubernetes.io/aws-weight: "100"
+  labels:
+    app: make-recall-decision-ui
+  name: make-recall-decisions-redirect-old-url
+  namespace: make-recall-decision-preprod
+spec:
+  ingressClassName: default
+  rules:
+    - host: make-recall-decision-preprod.hmpps.service.justice.gov.uk
+      http:
+        paths:
+          - backend:
+              service:
+                name: make-recall-decision-ui
+                port:
+                  name: http
+            path: /(.*)
+            pathType: ImplementationSpecific
+  tls:
+    - hosts:
+        - make-recall-decision-preprod.hmpps.service.justice.gov.uk
+      secretName: make-recall-decision-cert-old
```

Comment:
can u update the secret name to match the one in the certificate

Reply:
done ;)